src/plugins/intel_gna/src/transformations/convert_matmul_to_pointwise_convolution.cpp

```diff
@@ -49,9 +50,9 @@ static std::tuple<bool, uint32_t, uint32_t, uint32_t> VerifyAndGetConvParams(
     const uint32_t width = input1_shape.front();
     const uint32_t in_channels = input2_shape.back();
     const uint32_t out_channels = input2_shape.front();
-    if (input1_shape.front() <= limitations::affineMaxBatchSize ||
-        out_channels % limitations::convFiltersNumDivider != 0 || out_channels > limitations::convMaxFiltersNum ||
-        in_channels > limitations::convFilterMaxSize) {
+    if (input1_shape.front() <= Limitations::kAffineMaxBatchSize ||
```

Comment:
if it is used in more than one place please create method for this.

Reply:
out of PR scope